users/management/commands/update_site_name.py

```
@@ -0,0 +1,62 @@
+                    f'Site with ID {settings.SITE_ID} does not exist. '
+                    'Please run migrations first:\n'
+                    '  python manage.py migrate sites'
+                )
+            )
+        except AttributeError:
+            self.stdout.write(
+                self.style.WARNING(
+                    'PORTAL_NAME not found in settings. Using default "CinemataCMS"'
+                )
+            )
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)